packages/components/src/mobile/readable-content-view/index.native.js

```diff
@@ -62,6 +70,9 @@ const ReadableContentView = ( { align, reversed, children, style } ) => {
 						: styles.centeredContent,
 					style,
 					styles[ align ],
+					parentWidth && {
+						maxWidth: parentWidth + 2 * MARGIN,
```

Comment:
Hmmm, I'm not sure if it should be 16 in every case, maybe the `marginHorizontal` prop should be used? Also, I think you could pass `maxWidth` in the `style` prop from block-list-item

Reply:
Using `marginHorizontal` should work

Reply:
I've decided to pass max width tho